app/presenters/wpcc/period_contribution_presenter.rb

```diff
@@ -1,16 +1,39 @@
 module Wpcc
   class PeriodContributionPresenter < SimpleDelegator
-    delegate :t, to: :view_context
+    delegate :t, :number_to_currency, to: :view_context
 
     attr_reader :object, :view_context
 
     def initialize(object, args = {})
       super(object)
       @view_context = args[:view_context]
+      @object = object
     end
 
     def title
       t("wpcc.results.period_title.#{name}")
     end
+
+    def employee_contribution
+      formatted_contribution(object.employee_contribution)
+    end
+
+    def employer_contribution
+      formatted_contribution(object.employer_contribution)
+    end
+
+    def tax_relief
+      "(includes tax relief of #{formatted_contribution(object.tax_relief)})"
```

Comment:
Could you leave a note on the TP or somewhere to ensure that this part will be picked up for internationalisation when the Welsh locales are ready